manifesto/personas.md

```diff
@@ -0,0 +1,63 @@
+# Kyma Personas
+
+The following personas are fictional characters, which we defined in order to represent the different user types that might use Kyma. Keep in mind those personas when contributing to Kyma as they will help you to understand Kyma users' needs and goals (GAINS :green_heart:) and major pain points (PAINS :broken_heart:).
+
+1. [Matthew](#matthew---cloud-native-software-engineer)
+2. [Nicolas](#nicolas---tech-lead)
+
+## Matthew - Cloud Native Software Engineer
+
+![Mathew](assets/matthew.png)
+
+|                             |                                                                                                 |
+| --------------------------- | ----------------------------------------------------------------------------------------------- |
+| Name                        | **Matthew**                                                                                     |
+| Age                         | 26                                                                                              |
+| Ocupation                   | Cloud Native Engineer at **PureJoy**                                                            |
+| Characteristic              | Open source ethusiast and contributor. You will meet him at the webinars and community meetups. |
+| K8S experience (1-6)  | :book: :book: :book: :book: :book: :book:                                                       |
+| Accountability/Decisiveness (1-6) | 💪  💪                                                                                  |
+| Slogans                     | _Is there a CLI for that?_, _Can I automate it?_                                                |
```

Comment:
```suggestion
| Slogans                     | _Is there a CLI for that?_
 _Can I automate it?_                                                |
```

Reply:
Instead of a comma, I'd move the second slogan to the next line.

Reply:
Just in case you need it in the future: to do what you have in mind, you need to use `<br>` instead of plain `Enter/Return`.
```suggestion
| Slogans                     | _Is there a CLI for that?_ <br> _Can I automate it?_                                                |
```